core/src/Cuda/Kokkos_Cuda_Graph_Impl.hpp

```diff
@@ -199,6 +199,9 @@ struct GraphImpl<Kokkos::Cuda> {
         m_execution_space, _graph_node_kernel_ctor_tag{},
         aggregate_kernel_impl_t{});
   }
+
+  cudaGraph_t get_cuda_graph() const { return m_graph; }
+  cudaGraphExec_t get_cuda_graph_exec() const { return m_graph_exec; }
```

Comment:
No preconditions?

Reply:
I am not sure to see a good reason for precondition here.

Why would it be disallowed for an advanced user to query the `m_graph_exec` at anytime ?

Reply:
I guess we can close this one. There is no precondition in the `GraphImpl`, but there are in the `Graph`.